src/components/Contacts/Contacts.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['React' is defined but never used.](https://app.codacy.com/gh/WebHeroSchool/TODO-Olga/pullRequest?prid=5555083)